OpenRA.Mods.Common/Widgets/ObserverSupportPowerIconsWidget.cs

```diff
@@ -41,6 +42,14 @@ public ObserverSupportPowerIconsWidget(World world, WorldRenderer worldRenderer)
 			this.worldRenderer = worldRenderer;
 			clocks = new Dictionary<string, Animation>();
 			icon = new Animation(world, "icon");
+
+			// Timers should be synced to the effective game time, not the playback time.
+			GameSpeed speed;
+			var gameSpeeds = Game.ModData.Manifest.Get<GameSpeeds>();
```

Comment:
This is already enforced everywhere else. You can't define a mod without it. https://github.com/OpenRA/OpenRA/pull/9255